src/taco-livereload/lib/utils/Patcher.js

```diff
@@ -31,63 +28,50 @@ function Patcher(projectRoot, platforms) {
  *    - Removes ATS (App Transport Security) in the case of iOS9+
  *    - Copies the homePage to the mobile app
  *    - Changes the start page of the app to the homePage
- * 
+ *
  * HomePage Role:
- *    - The homePage allows us to ping the BrowserSync server 
+ *    - The homePage allows us to ping the BrowserSync server
  *    - If we can't connect to the BrowserSync server, we display a meaningful error message to the user
  *    - If we can successfully connect to the BrowserSync server, we proceed with loading the app
  */
 Patcher.prototype.patch = function (serverUrl) {
     var self = this;
+    self.removeCSP();
     return promiseUtils.Q_chainmap(self.platforms, function (plat) {
-        var platWWWFolder = multiPlatforms.getPlatformWWWFolder(plat);
-        var platformIndexLocal = path.join(self.projectRoot, platWWWFolder, self.startPage);
-        var cspRemover = new CSPRemover(platformIndexLocal);
-        return cspRemover.Remove().then(function () {
-            var atsRemover = new ATSRemover(self.projectRoot, plat);
-            return atsRemover.Remove();
-        }).then(function () {
-            var platformIndexUrl = url.resolve(serverUrl, path.join(multiPlatforms.getPlatformWWWFolder(plat), self.startPage));
-            return copyHomePage(self.projectRoot, plat, platformIndexUrl);
-        }).then(function (homePage) {
-            return helpers.ChangeStartPage(self.projectRoot, plat, homePage); 
-        });
+        if (plat == 'ios')
```

Comment:
Oh I see that this is actually a js file and so the tslint wouldn't pick it up anyway. Still good to try and be consistent I think